recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -152,6 +152,7 @@ HAS_GBM_mx8 = "true"
 GLES3_HEADER_REMOVALS        = "gl31.h gl32.h"
 GLES3_HEADER_REMOVALS_mx8m   = "gl32.h"
 GLES3_HEADER_REMOVALS_mx8mm  = "gl31.h gl32.h"
+GLES3_HEADER_REMOVALS_mx8mp  = "gl32.h"
```

Comment:
Not necessary because of line 153.

Reply:
Right, missed the _MACHINEOVERRIDES_. Thanks for pointing this out!